models/Asset/Thumbnail/ImageThumbnailTrait.php

```diff
@@ -429,4 +430,20 @@ private function getFilename(): string
 
         return basename($pathReference['src']);
     }
+
+
+    /**
+     * @description Returns path for thumbnail image in a given file format
+     * @param string $format
+     * @return self
+     */
+    public function getAsFormat(string $format): self
+    {
+        $config = $this->getConfig() ?? new Config();
+        $config->setFormat($format);
+
+        $this->config = $config;
+
+        return $this;
```

Comment:
```suggestion
        $thumb = clone $this;
        $config = clone $thumb->getConfig();
        $config->setFormat($format);
        $thumb->setConfig($config); 

        return $thumb ;
```
We need to clone the instance, otherwise this will change the behavior of the original object.